subcommand/server-acl-init/rules.go

```diff
@@ -174,12 +183,12 @@ operator = "write"
 agent_prefix "" {
   policy = "read"
 }
+node_prefix "" {
+  policy = "write"
+}
```

Comment:
What is the difference between these two placements? As far as I can tell the below placement means the node:write policy will apply to all namespaces with `namespace_prefix` being `""`.

Reply:
Yeah I agree but it's not allowed in there, it errors out.

Reply:
`rpc error making call: rpc error making call: Invalid namespace acl policy: namespace policies may not grant write privileges to nodes`

Reply:
Do you mean specifically the `write` policy? We have examples in [our docs](https://www.consul.io/docs/acl/acl-rules.html#namespace-rules-enterprise) that show node:read policy, so I'm pretty sure we can put `node` polices under `namespace_prefix`. Am I missing something?

Reply:
Oh strange, when I was writing my comment I didn't see your second response. That makes sense.